lib_a/conanfile.py

```diff
@@ -44,7 +44,7 @@ def _configure_cmake(self):
 
         self._cmake.definitions["BUILD_TESTING"] = self.options.with_tests
 
-        self._cmake.configure(source_folder="src")
+        self._cmake.configure(source_folder=self.source_folder)
```

Comment:
```suggestion
        self._cmake.configure(r)
```
 I think this is the default setting, right?

Reply:
probably